### DISCUSSION TITLE
How to add border to bars and arcs

### DISCUSSION THREAD

**Discussion by mcp292** [vega/altair#2989] [Q&A] [answered]
# Question
Is there a way to add a black border to bar and arc charts similar to [this](https://en.wikipedia.org/wiki/Pie_chart):
![bordered pie chart](https://user-images.githubusercontent.com/49123398/227609542-f3a7fd84-00cf-4085-98ae-2bc668807414.png)

# [MWE](https://altair-viz.github.io/gallery/pie_chart.html)
```py
import pandas as pd
import altair as alt

source = pd.DataFrame({"category": [1, 2, 3, 4, 5, 6], "value": [4, 6, 10, 3, 7, 8]})

alt.Chart(source).mark_arc().encode(
    theta=alt.Theta(field="value", type="quantitative"),
    color=alt.Color(field="category", type="nominal"),
)
```

**Comment by jonmmease** [comment, marked as answer]
Yes, you can use the `stroke` and `strokeWidth` arguments to `mark_arc`:

```python
import pandas as pd
import altair as alt

source = pd.DataFrame({"category": [1, 2, 3, 4, 5, 6], "value": [4, 6, 10, 3, 7, 8]})

alt.Chart(source).mark_arc(stroke="black", strokeWidth=3).encode(
    theta=alt.Theta(field="value", type="quantitative"),
    color=alt.Color(field="category", type="nominal"),
)
```
![visualization (1)](https://user-images.githubusercontent.com/15064365/227610029-2a759900-01c5-481a-ae55-c629aab9ca22.png)

Hope that helps!

**Reply by mcp292** [reply]
Outstanding thank you!
